django/test/runner.py

```diff
@@ -97,25 +97,51 @@ def debug(self, error):
         pdb.post_mortem(traceback)
 
 
-class RemoteTestResult:
+class DummyList:
     """
-    Record information about which tests have succeeded and which have failed.
+    Dummy list class for faking storage of results in unittest.TestResult. We
+    do this because RemoteTestResult replaces the way of recording.
+    """
+    __slots__ = []
 
-    The sole purpose of this class is to record events in the child processes
-    so they can be replayed in the master process. As a consequence it doesn't
-    inherit unittest.TestResult and doesn't attempt to implement all its API.
+    def append(self, item):
+        pass
```

Comment:
not sure if it's better to just replace with a `mock.Mock()`.

Reply:
I don't think it would make a difference in memory efficiency. Seems fine to keep it as it is.

Reply:
On second thoughts, it would be less efficient that a list. Mocks record a list of all calls made against them.

I also admit that replacing the lists is even a *bit* of a premature optimization. In large projects I imagine there could be thousands of test results stored in memory, which could translate at best to a few megabytes of memory usage.